dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -488,6 +489,10 @@ private String getXML(List<Contentlet> cons, HttpServletRequest request, HttpSer
 			if(s.getStructureType() == Structure.STRUCTURE_TYPE_WIDGET && "true".equals(render)) {
 				m.put("parsedCode",  WidgetResource.parseWidget(request, response, c));
 			}
+			
+			if (s.getStructureType() == Structure.STRUCTURE_TYPE_HTMLPAGE){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "STRUCTURE_TYPE_HTMLPAGE"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

Reply:
This can be changed to use the new approach:
```
if (BaseContentType.HTMLPAGE.equals(content.getContentType().baseType())) {
```